build.yaml

```diff
@@ -928,14 +935,16 @@ job_type:
   # http://build.clusterhq.com/builders/flocker-docs
   run_sphinx:
     run_sphinx:
-      on_nodes_with_labels: 'aws-centos-7-T2Small'
+      on_nodes_with_labels: 'aws-ubuntu-trusty-T2Small'
       with_steps:
         - { type: 'shell',
             cli: [ *hashbang, *add_shell_functions,  *setup_pip_cache,
                    *cleanup, *setup_venv, *setup_flocker_modules,
-                   *setup_aws_env_vars, *run_sphinx ]
+                   *install_aws_cli, *run_sphinx,
+                   *get_s3_creds_for_bucket_staging_docs,
+                   *upload_new_docs_html_to_s3_staging_docs]
           }
-      timeout: 30
+      timeout: 10
 
```

Comment:
the job only takes 3 minutes to run, so not point in defining a timeout 10 times longer than that.